content/guides/caching.md

```diff
@@ -4,284 +4,358 @@ sort: 11
-    </script>
-  </head>
-  <body>
-  </body>
-</html>
+Running another build, we would expect only our `main` bundle's hash to change, however...
+
+``` bash
+Hash: d38a06644fdbb898d795
+Version: webpack 3.3.0
+Time: 1445ms
+                          Asset       Size  Chunks                    Chunk Names
+ vendor.a7561fb0e9a071baadb9.js     541 kB       0  [emitted]  [big]  vendor
+   main.b746e3eb72875af2caa9.js    1.22 kB       1  [emitted]         main
+runtime.1400d5af64fc1b7b3a45.js    5.85 kB       2  [emitted]         runtime
+                     index.html  352 bytes          [emitted]
+   [1] ./src/index.js 421 bytes {1} [built]
+   [2] (webpack)/buildin/global.js 509 bytes {0} [built]
+   [3] (webpack)/buildin/module.js 517 bytes {0} [built]
+   [4] ./src/print.js 62 bytes {1} [built]
+   [5] multi lodash 28 bytes {0} [built]
+    + 1 hidden module
 ```
 
-At the end of the day, the hashes for the files should be based on the file content. For this use [webpack-chunk-hash](https://github.com/alexindigo/webpack-chunk-hash) or [webpack-md5-hash](https://github.com/erm0l0v/webpack-md5-hash).
-
-So the final `webpack.config.js` should look like this:
-
-```js
-var path = require("path");
-var webpack = require("webpack");
-var ChunkManifestPlugin = require("chunk-manifest-webpack-plugin");
-var WebpackChunkHash = require("webpack-chunk-hash");
-
-module.exports = {
-  entry: {
-    vendor: "./src/vendor.js", // vendor reference file(s)
-    main: "./src/index.js" // application code
-  },
-  output: {
-    path: path.join(__dirname, "build"),
-    filename: "[name].[chunkhash].js",
-    chunkFilename: "[name].[chunkhash].js"
-  },
-  plugins: [
-    new webpack.optimize.CommonsChunkPlugin({
-      name: ["vendor", "manifest"], // vendor libs + extracted manifest
-      minChunks: Infinity,
-    }),
-    new webpack.HashedModuleIdsPlugin(),
-    new WebpackChunkHash(),
-    new ChunkManifestPlugin({
-      filename: "chunk-manifest.json",
-      manifestVariable: "webpackManifest",
-      inlineManifest: true
-    })
-  ]
-};
+... we can see that all three have. This is because the [`module.id`](/api/module-variables#module-id-commonjs-) of each is incremental by default, so...
+
+- The `main` bundle changed because of it's new content.
+- The `vendor` bundle changed because it's `module.id` was changed.
+- And, the `runtime` bundle changed because it now contains a reference to a new module.
+
+The first and last are expected -- it's the `vendor` hash we want to fix. Luckily, there are two plugins that can help us out with this dilemma. First, the [`NamedModulesPlugin`](/plugins/named-modules-plugin) which the path to the module rather than a numerical ID. While this plugin is useful during development for easier to read output, it does take a bit longer to run. The second option is the [`HashedModuleIdsPlugin`](/plugins/hashed-module-ids-plugin), which is what we'll use as these examples are more targeted toward production builds:
```

Comment:
`for easier to read output` => `for simpler output` or `for the output that is easier to read`?